test-deployment.yml

```diff
@@ -3,13 +3,6 @@ kind: Deployment
 metadata:
   name: example
   namespace: production
-  annotations:
```

Comment:
we remove annotations?

Reply:
Now we do now that annotations arent used to generate imageDescriptions for spinnaker. They can still be used though if the deployment really wants.